### DISCUSSION TITLE
man!小登入坑崩坏三合适吗？

### DISCUSSION THREAD

**Discussion by AAAduck** [share121/inter-knot#1783] [其他]
小登新手池子出死生是不是可以重开了？
大概一周多一点，55级。
![Image_1723474914083](https://github.com/user-attachments/assets/b4de712a-481a-47ff-a422-ef3d36a4bf9b)

**Comment by 1451374815** [comment]
只看剧情，别深度入坑，赶紧跑

**Reply by AAAduck** [reply]
但是我喜欢里面的角色，想收集，但是剧情确实太长了，不过感觉不是很肝，自己游戏里面过一遍，然后再在b站看

**Comment by jiaopanda** [comment]
老登们说的深度入坑就是打战场+深渊，乐土的话.....emmmm！！！
这些都随意吧，你只过剧情什么的都好，就是崩三的剧情跨度比较割裂，毕竟剧情也重置了一两次，前大半还是要结合漫画来看的，后面从女王降临开始，就有大部分的过场CG来帮助你了解具体剧情

**Comment by kiana-codek423** [comment]
强烈建议只过剧情！！崩崩崩数值膨胀很严重，一个角色的圣痕（相当于圣遗物 也是要抽的，除非欧皇氪佬不然很难凑齐
不打深渊乐土是不影响剧情观看的，会给你试用角色

**Comment by 566001** [comment]
别玩崩坏三，建议你玩崩坏学院2

**Comment by AOEEVU** [comment]
强烈建议入坑蹦蹦

**Comment by yue0ying** [comment]
建议入坑崩2

**Comment by ruanmiaomiao** [comment]
死生好用啊






等等？新人？快跑！！！！不要回头！！！！
